frontend/Dockerfile.frontend

```diff
@@ -0,0 +1,6 @@
+FROM node:18
+WORKDIR /app
+COPY package*.json ./
+RUN npm install
```

Comment:
This project uses `yarn` instead of `npm`, so `npm` should be replaced with `yarn`

Install yarn before this using the `corepack enable` command, too